r/R/get_expression_cellsets.R

```diff
@@ -0,0 +1,77 @@
+#' Get expression genes for pathway analysis
+#'
+#' @param req request parameters
+#'   e.g. list(
+#'       body = list(
+#'         list(geneName = 'Gene1', comparisonType = 'greaterThan', thresholdValue = 0.5),
+#'         list(geneName = 'Gene2', comparisonType = 'lessThan', thresholdValue = 0.5),
+#'         ...
+#'       )
+#'   )
+#' @param data SeuratObject
+#'
+#' @return vector of cell ids.
+#' @export
+#'
+
+
+
+getExpressionCellSets <- function(req, data) {
+  new_cell_set_data <- getExpressionCellSetIDs(req$body$genesConfig, data)
+  keep_ids <- unname(new_cell_set_data$keep_ids)
+  cell_set_name <- new_cell_set_data$cell_set_name
+  new_cell_set <- list(key = uuid::UUIDgenerate(use.time = TRUE), name = cell_set_name, rootNode = FALSE, color = sample(data@misc$color_pool, 1), cellIds = keep_ids)
+  cell_set_class_key <- "scratchpad"
+  config <- req$body$config
+
+  insert_set_child_through_api(new_cell_set, config$apiUrl, config$experimentId, cell_set_class_key, config$authJwt)
```

Comment:
camelCase and long line

```suggestion
insertSetChildThroughApi(
  new_cell_set,
  config$apiUrl,
  config$experimentId,
  cell_set_class_key,
  config$authJwt
)
```